README.md

```diff
@@ -65,6 +67,43 @@ echo 'GITHUB_TOKEN=<your api token>' >> ~/.config/bonfire/env
 
 **Please note**: When using the Github token (classic), it **must have** the `read:project` scope for the template fetching to work!
 
+## Using Docker
+
+We provide a Docker image at quay.io/cloudservices/bonfire. The image wraps a Python virtual environment with **crc-bonfire** installed on it.
+
+In order to use Bonfire you have to provide either a valid KUBE_CONFIG or a URL and a TOKEN to connect to the target Openshift cluster.
+
+The container's entrypoint already invokes the **bonfire** CLI with the arguments passed to the container, so you should be able to run any bonfire command you'd typically run by using:
+
+```
+podman run -it --rm quay.io/cloudservices/bonfire version
+```
+
+### Using credentials
+
+The Bonfire image can use the `OC_LOGIN_SERVER` and `OC_LOGIN_TOKEN` variables to connect to an Openshift cluster.
+
+```
+podman run -it --rm -e OC_LOGIN_SERVER=https://api.openshift.example.com:6443 \
+    -e OC_LOGIN_TOKEN=some-api-token \
+    quay.io/cloudservices/bonfire namespace list
+```
+
+It can also make use of a Kube config file by mounting it on the container's HOME:
+
+```
+podman run -it --rm -v $HOME/.kube/config:/opt/bonfire/.kube/config:Z,U \
+    quay.io/cloudservices/bonfire namespace list
```

Comment:
After some reading, I think this is how we can run without messing up the local kube config permissions:
```
--userns=keep-id:uid=1000,gid=1000
```
(using 1000 since that is the uid and gid of the user you set up in your Dockerfile)

Seems to be working for me. Read this GH issue and the last comment [here](https://github.com/containers/podman/issues/7052) -- the option is now documented [here](https://docs.podman.io/en/v4.4/markdown/options/userns.container.html#userns-mode) 

The other thing that works for me is running with `-u root`. If you do that, you need to add `ENV KUBECONFIG=/opt/bonfire/.kube/config` to the Dockerfile. [This article](https://www.tutorialworks.com/podman-rootless-volumes/ ) was helpful explaining the user namespaces, but it didn't mention the `--userns` options, those are probably newer :)

Reply:
I think running with root is what happens when you use regular Docker or force root on podman. I want to believe rootless podman is where we need to strive to whenever possible. 

Thanks for sharing that Podman Issue - that was revealing, it was a great finding. So reading it, it basically confirms on Docker or running Podman with root you get a consistent behavior, but if you try running rootless podman you face this issue (and someone in the thread points out you'll face the same too if running rootless Docker). 

So it looks like, specially considering Nov 22' entry mentions this new feature, all we have to do when using Podman is to use the `--userns=keep-id` option. 

I tested it and works like a charm, so I think you found the solution ! Awesome ! :rocket: 

My only concern is that is "relatively new" feature in podman. I'm on Fedora 39, podman version I have is 4.8.3 

For the moment, I'm gonna support using this since:
- It works like a charm in rootless podman
- It allows to suggest a way to share the Kube config context with the container, which makes changes to the context that persist out of the container without messing the file's permissions (so subsequent local `oc` or `kubectl` commands should be okay).

I like this. Thanks again, great job Brandon !